orbital-mechanics/Lecture12/Lecture12.ipynb

```diff
@@ -0,0 +1,280 @@
+{
```

Comment:
<ol><li>Second level title for the orbit.</li><li>Something weird is happening with the html; is this just on here? Please clarify.</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angadhn/SpacecraftDynamics/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Is it the ' Note that there are (Y + 1) sidereal days in 1 year.' part? if so, I can make the adjustments to avoid the issue with rendering the html.</p>